src/components/auth/LoginForm.jsx

```diff
@@ -129,7 +131,6 @@ const LoginForm = () => {
                     className="btn-secondary txt-s w-80"
                     onClick={(e) => {
                       e.preventDefault();
```

Comment:
ok you did it here so delete above as double prevention to same event not needed

Reply:
will do